pkg/local_object_storage/pilorama/boltdb.go

```diff
@@ -71,12 +78,9 @@ func (t *boltForest) SetMode(m mode.Mode) error {
 	if t.mode == m {
 		return nil
 	}
-	if t.mode.ReadOnly() == m.ReadOnly() {
-		return nil
-	}
 
 	err := t.Close()
```

Comment:
not related to that PR but why is it the `t` receiver for `boltForest`? i thought that i was looking at a test code for a second

Reply:
`t` for `forest of Trees` (initially it was all about `trees`, so `t` stuck)